frontend/src/views/supp-allowances/SupplementarySubmitView.vue

```diff
@@ -157,7 +165,9 @@ export default {
     },
     isSupportComplete() {
       const model = this.getModel(SUPPLEMENTARY_TYPES.SUPPORT)
-      if (!model?.supportFundingModel.includes(this.SUPPORT_CHECKBOX_LABELS.find((item) => item.label === 'Other').value)) {
+      if (!this.currentOrg.hasInclusionPolicy && model) {
```

Comment:
I think a warning banner is good enough for missing inclusion policy. The "missing required info" alert should be only for the incomplete Supplementary form (not for missing inclusion policy)
![image](https://github.com/user-attachments/assets/34367e0e-9e0a-4650-8f82-a5ca74a8e94d)

Reply:
done!